content/posts/rollout-of-our-new-dashboard.rst

```diff
@@ -0,0 +1,72 @@
+:title: Rollout of our new dashboard
+:date: 2024-11-11
+:description: Our new dashboard will begin rolling out to users.
+              Learn what to expect and how it will affect you and your project.
+:category: Changelog
+:tags: dashboard
+:authors: Anthony Johnson
+:status: published
+:image: /images/new-dashboard.png
+:image_credit: Reviewing build output for a project in our new dashboard
+
+For the last year, you may have seen notifications and hints asking you to try
+out our new dashboard. This new dashboard has been a long term project to
+completely replace our still original user interface with one that is more
+consistent and allows us to more easily build and maintain complex features.
+
+During the next few months, we will begin the transition to our new dashboard
+and will move away from our current, *but soon to be legacy*, dashboard.
+
+If you are not yet using our new dashboard, here is what you need to know:
+
+The new dashboard will start to become the default dashboard
+    After this change, users logging in from our website will be logging into
+    our new dashboard. *Users will still be able to switch back to the legacy
+    dashboard if they need to*.
+
+    - On `Read the Docs Community`_ this change will go into effect on **Nov 12th**
+    - On `Read the Docs Business`_ this change will go into effect on **Dec 10th**
+
+The legacy dashboard will be available until Mar 1st, 2025
```

Comment:
Do we want to define a date here before we've migrated over? Feels a little ambitious, but probably good to have a deadline, and just do the work to make sure we hit it.

Reply:
Yeah I felt a date was fine here. The only thing we're waiting on in this period is for users to switch their daily usage to the new dashboard -- we don't need users to make content changes and we won't have any migration tasks left after Dec 10th.

We can also always extend this timeframe if users are stuck using the old dashboard for some reason.

Reply:
I'll call the date Mar 11th, so 3 months even from Dec 10th.

```suggestion
The legacy dashboard will be available until Mar 11th, 2025
```